firebase-vscode/package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "firebase-vscode",
+  "displayName": "firebase-vscode",
+  "publisher": "firebase",
+  "description": "",
```

Comment:
nit: Anything good we can put in here?

Reply:
This reminds me, I just put the name and display name as a placeholder and probably need to talk to the wider team and PM about that.